app/javascript/components/NewsLetter.jsx

```diff
@@ -1,70 +1,64 @@
 import React, { useState, useRef } from "react";
 import axios from "axios";
-
+import validator from "validator";
 import {
   Button,
-  Heading,
-  TextInput,
   Text,
   Box,
   ProgressIndicator,
-  VStack,
-  Stack,
+  Link,
   useNYPLBreakpoints,
-  useColorModeValue,
+  NewsletterSignup,
 } from "@nypl/design-system-react-components";
 
 export default function NewsLetter() {
   const [message, setMessage] = useState("");
   const [email, setEmail] = useState("");
   const [buttonDisabled, setButtonDisabled] = useState(false);
-  const [isInvalid, setIsInvalid] = useState(false);
+  const [isInvalidEmail, setIsInvalidEmail] = useState(false);
   const [successFullySignedUp, setSuccessFullySignedUp] = useState(false);
+  const [confirmationHeading, setConfirmationHeading] = useState("");
   const { isLargerThanMobile } = useNYPLBreakpoints();
-  const newsLetterbg = useColorModeValue("ui.bg.default", "dark.ui.bg.default");
   const newsLetterMsgRef = useRef(null);
+  const [view, setView] = React.useState("form");
 
   const handleNewsLetterEmail = (event) => {
     setEmail(event.target.value);
-    setIsInvalid(false);
+    setIsInvalidEmail(false);
   };
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    setButtonDisabled(true);
-
+    if (!validator.isEmail(email)) {
+      setIsInvalidEmail(true);
+      setView("form");
+      return;
+    } else {
+      setView("submitting");
+    }
     axios
       .get("/news_letter/index", {
         params: {
           email: email,
         },
       })
       .then((res) => {
-        setMessage(res.data.message);
-        setTimeout(() => {
-          const nlTextInputlement = document.getElementById(
-            "news-letter-text-input"
-          );
-          if (nlTextInputlement) {
-            nlTextInputlement.focus();
-          }
-        }, 1);
-
         if (res.data.status === "success") {
-          setIsInvalid(false);
           setSuccessFullySignedUp(true);
-          // Set focus to the element
-          setTimeout(() => {
-            const newsLetterElement = document.getElementById(
-              "news-letter-msg-box"
-            );
-            if (newsLetterElement) {
-              newsLetterElement.focus();
-            }
-          }, 1);
+          setView("confirmation");
+          setConfirmationHeading("Thank you for signing up to the Newsletter!");
```

Comment:
Update the message to the following.
```
Thank you for signing up for our newsletter!
```

Reply:
@bigfishdesign13  Yes Marty New letetr signup is slow. We are checking from google sheets.